scripts/tools/eslint-plugin-prettier-internal-rules/massage-ast-parameter-names.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+const MESSAGE_ID = "massage-ast-parameter-names";
+
+const massageAstFunctionSelector = [
+  "FunctionDeclaration",
+  "[async!=true]",
+  "[generator!=true]",
+  '[id.type="Identifier"]',
+  '[id.name="clean"]',
+].join("");
+
+const getVariableIdentifiers = ({ identifiers, references }) => [
+  ...new Set([
+    ...identifiers,
+    ...references.map(({ identifier }) => identifier),
+  ]),
+];
+
+module.exports = {
+  meta: {
+    type: "suggestion",
+    docs: {
+      url: "https://github.com/prettier/prettier/blob/main/scripts/tools/eslint-plugin-prettier-internal-rules/massage-ast-parameter-names.js",
+    },
+    messages: {
+      [MESSAGE_ID]:
+        "The {{name}} node parameter '{{original}}' should be named '{{name}}'.",
+    },
+    fixable: "code",
+    schema: {
+      type: "array",
+      uniqueItems: true,
+    },
+  },
+  create: (context) =>
+    Object.fromEntries(
+      ["original", "clone"].map((name, index) => [
```

Comment:
The variable name clone sounds like a function, so cloned or clonedNode is preferable.